doc/pmon/pmon-sensormon.md

```diff
@@ -0,0 +1,260 @@
+	VP0P75_CORE_NPU0         750 mV        852       684             872            664      False  20230204 11:35:21
+	VP0P75_CORE_NPU1         750 mV        852       684             872            664      False  20230204 11:35:21
+	VP0P75_CORE_NPU2         750 mV        852       684             872            664      False  20230204 11:35:22
+	
+	...
+	
+	root@sonic:/home/cisco# show platform current
+	        Sensor        Current    High TH    Low TH    Crit High TH    Crit Low TH    Warning          Timestamp
+	--------------  -------------  ---------  --------  --------------  -------------  ---------  -----------------
+	POL_CORE_N0_I0       25000 mA      30000     18000           28000          15000      False  20230212 11:18:28
+	POL_CORE_N0_I1       21562 mA      30000     18000           28000          15000      False  20230212 11:18:28
+	POL_CORE_N0_I2       22250 mA      30000     18000           28000          15000      False  20230212 11:18:28
+
+
+	
+
+#### Configuration and Management
+
+At this point, there is no configuration requirement for this feature. 
+
+If the SensorMon daemon is not desired to be run in the system, an entry can be added to  pmon_daemon_control.json to exclude it from running in the system.
+
+It is advised to monitor the sensor alarms and use that to debug and identify any issues in the system. In the event, a sensor crosses a high or low threshold, syslogs will be raised indicating the alarm.
+
+	FJul 27 08:26:32.561330 sonic WARNING pmon#sensormond: High voltage warning: VP0P75_CORE_NPU2 current voltage 880mV, high threshold 856mV
+
+The alarm condition will be visible in the CLI ouputs for sensor data and system health.
+
+	e.g 
+	root@sonic:/home/cisco# show platform voltage
+	          Sensor        Voltage    High TH    Low TH    Crit High TH    Crit Low TH    Warning          Timestamp
+	----------------  -------------  ---------  --------  --------------  -------------  ---------  -----------------
+	VP0P75_CORE_NPU2         880 mV        720       684             720            664       True  20230204 11:35:22
+	
+	root@sonic:/home/cisco# show system-health detail
+	System status summary
+  	...
+  	  Hardware:
+       Status: Not OK
+       Reasons: Voltage sensor VP0P75_CORE_NPU2 measurement 880 mV out of range (679,856)
+    	...
+    	VP0P75_CORE_NPU2            Not OK    voltage
+    	...
+
+
+##### Platform Sensors Configuration
+
+Sensormond will use the platform APIs for retrieving platform sensor information. However, for platforms with only file-system/sysfs based drivers, a simple implementation is provided wherein the platform can specify the sensor information for the board and any submodules (such as fabric cards) in a data file and Sensormond can use that for finding sensors and monitoring them. 
+
+The file system/Sysfs based platform sensor information can be provided using a yaml file. The yaml file shall have the following format.
```

Comment:
@bmridul what is the location of the yaml file for platform? Is at HWSKU level or platform level?

Reply:
Its at the platform level -  /usr/local/etc/sensors.yaml
Pls see https://github.com/sonic-net/sonic-platform-common/pull/426